### PR TITLE
Fixes #34529 - Lint failure due to uncontrolled eslint-plugin

### PR DESCRIPTION
**Pull request by Andrewgdewar**
#### What are the changes introduced in this pull request?

The lint package, eslint-plugin-react, was updated from: "7.28.0" to "7.29.1" due to the lack of a committed package-lock in foreman.

Related changelog for eslint-plugin-react [here](https://github.com/yannickcr/eslint-plugin-react/blob/master/CHANGELOG.md).

This fixes some breaking lint changes introduced by that update.

#### Considerations taken when implementing this change?
- I thought it prudent to use the updated rules instead of just turning them off for a solution. 
- If this becomes problematic or slows down the dev process, one can simply change two lines in the eslintrc.js file.

`
'indent': 'off',
'react/forbid-prop-types': 'off',
`

#### What are the testing steps for this pull request?

- Check if the Katello lint and react tests pass on CI.
- ???? 
- Profit

### CODE REVIEW

File: webpack/components/WithOrganization/withOrganization.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-indent */
```
Comment (sjha4) [inline webpack/components/WithOrganization/withOrganization.js:1]: Do we need this even after ignoring all the JSX* nodes in the lint rule?

Comment (Andrewgdewar) [reply webpack/components/WithOrganization/withOrganization.js:1]: Yes. In this one case it is needed. I have no idea why.

Comment (Andrewgdewar) [reply webpack/components/WithOrganization/withOrganization.js:1]: First question (longer answer), there are probably a hundred "ignoredNodes" options for the indent lint rule. I'm not sure which one was being applied in this file. It was behaving differently than the rest. Unsure as to why. 

Second question: 
 eslint --fix

**Review by jeremylenz — APPROVED** [review]
APJ :+1:

**Pull request merged**